source/_posts/2021-03-06-android-2021-q1-releases.markdown

```diff
@@ -0,0 +1,95 @@
+
+Some of you may have noticed that starting in 2021.1 the app was showing a prompt before a user was able to interact with the Home Assistant frontend. After 2021.1 we had received a lot of feedback from users about the intrusive pop-up. Starting in 2021.2 this pop-up has been converted to a persistent notification that has its own notification channel allowing the user to fully control how it is displayed on the device. This means you can turn off this notification channel like any other channel on your device. The new channel name for this notification is `Location disabled`. This persistent notification will automatically clear once location is enabled and the application has proper permissions. The notification will list which features are currently disabled.
+
+<p class='img'>
+<img src='/images/blog/2021-03-06-android-q1-releases/location_disabled.png' alt='Screenshot of the location disabled notification'></a>
+Screenshot of the location disabled notification.
+</p>
+
+## High Accuracy Mode
+
+A new feature was added in 2021.2 to allow users to get much faster location updates. This feature will cause additional battery drain. When enabled a persistent notification will be displayed containing some location data. This feature is an enhancement to the Background Location sensor and you can access it from the sensor settings screen. You can also control this feature via a new notification command to enable/disable it on the fly. You can learn more about this feature in the [docs](https://companion.home-assistant.io/docs/core/location#high-accuracy-mode).
+
+<p class='img'>
+<img src='/images/blog/2021-03-06-android-q1-releases/high_accuracy.png' alt='Screenshot of the high accuracy mode notification'></a>
+Screenshot of the high accuracy mode notification.
+</p>
+
+## New Sensors
+
+We have several new sensors to welcome to the app, all of which are disabled by default.
+
+- [Active Notifications](https://companion.home-assistant.io/docs/core/sensors#active-notification) - The total count of active notifications visible to the user. Attributes will include all notification data.
+
+- [App Data Sensors](https://companion.home-assistant.io/docs/core/sensors#app-data-sensors) - Sensors to determine how much data the app has used since the last device reboot.
+
+- [App Importance Sensor](https://companion.home-assistant.io/docs/core/sensors#app-importance-sensor) - A sensor to determine if the app is in the `foreground`, `background` or any other importance level.
+
+- [App Memory Sensor](https://companion.home-assistant.io/docs/core/sensors#app-memory-sensor) - A sensor to determine how much memory is used by the app.
```

Comment:
This is … a really odd sensor to have ? Why would anyone ever care besides developers?

Reply:
I agree it is a bit odd.  A lot of these app sensors were at first designed for toubleshooting then they also became useful for other aspects of the app.  In this case its probably more of a dashboard sensor :)